pkg/proxy/winkernel/proxier.go

```diff
@@ -1177,7 +1184,7 @@ func (proxier *Proxier) syncProxyRules() {
 			// Try loading existing policies, if already available
 			hnsLoadBalancer, err = hns.getLoadBalancer(
 				hnsEndpoints,
-				loadBalancerFlags{sessionAffinity: sessionAffinityClientIP, isIPv6: proxier.isIPv6Mode},
+				loadBalancerFlags{isDSR: svcInfo.dsr, sessionAffinity: sessionAffinityClientIP, isIPv6: proxier.isIPv6Mode},
```

Comment:
Would we want to also check proxier.isDSR for these other loadbalancer types?

Reply:
I was not planning on altering any functionality surrounding isDSR and preserveDIP, @madhanrm could you weigh in on this?

Reply:
Let's follow up in a separate PR to use proxier.isDSR for these loadbalancers as well.

Reply:
> Let's follow up in a separate PR to use proxier.isDSR for these loadbalancers as well.

would that follow-up PR be required to be in 1.19 as well? as noted, we are past code freeze

Reply:
No that is for 1.20. This PR is now only updating a dependency which contains a fix for Windows 2004

Reply:
@liggitt this is no longer in scope for this PR and v1.20. Originally was trying to implement externalTrafficPolicy local in winkernel kube-proxy in this PR, but we are going to hold off on that and the followup work mentioned here until v1.20. This PR is now just simply vendoring in a version check update from hcsshim

Reply:
ack